waku/waku.md

```diff
@@ -92,15 +91,28 @@ rate-limits = "[" limit-ip limit-peerid limit-topic "]"
 
 light-node = BIT
 
+pow-requirement-key = 0
```

Comment:
Keys could also be strings, but this seemed more 'idiomatic' in RLP. The only gotcha is that IME using numbered keys makes it seem like it is an ordered set, which is not the intent - its semantics are that of an unordered map.